contrib/gdk-pixbuf/loader.c

```diff
@@ -467,7 +467,7 @@ static gboolean avif_image_saver(FILE          *f,
     encoder->maxQuantizer = max_quantizer;
     encoder->minQuantizerAlpha = 0;
     encoder->maxQuantizerAlpha = alpha_quantizer;
-    encoder->speed = 8;
+    encoder->speed = 7;
```

Comment:
I recommend speed 6. I will merge this pull request first and let you test speed 6 and see if it is fast enough.